File: server/UniversityApp.Model/EstadoAcademico.cs
using System.Collections.Generic;
using UniversityApp.DB;

namespace UniversityApp.Model
{
    public class EstadoAcademico
    {
        public int IDAlumno { get; set; }
        public IEnumerable<Asignatura> AsignaturasPorCursar { get; set; }
        public IEnumerable<Asignatura> AsignaturasInscripto { get; set; }
        public IEnumerable<Asignatura> AsignaturasNoRegular { get; set; }
    }
}

File: client/university-app/src/app/model/estadoAcademico.ts
import {Asignatura} from './asignatura';

export class EstadoAcademico {
  IDAlumno: number;
  AsignaturasPorCursar: Asignatura[];
  AsignaturasInscripto: Asignatura[];
  AsignaturasNoRegular: Asignatura[];
}
